buildenv/jenkins/jobs/infrastructure/Cleanup-Nodes.groovy

```diff
@@ -179,6 +179,14 @@ timeout(time: TIMEOUT_TIME.toInteger(), unit: TIMEOUT_UNITS) {
                                         }
                                     }
 
+                                    // Cleanup zOS datasets 
+                                    if (nodeLabels.contains('sw.os.zos')) {
+                                        listcat = sh(script: "tso listcat | grep '${env.USER}' | cut -d. -f 2-", returnStdout: true).trim()
```

Comment:
Are you sure this works? It seems like it should but when I try `tso listcat | grep JENKINS` manually, it doesn't find anything although there is a dataset there.

Reply:
Trying something else `tso listcat | tee out` it outputs garbage, making me thing there is a ascii/ebcdic issue.

`tso listcat > out && grep JENKINS out` works.

Reply:
I was using fyrec50g

Reply:
Actually `sh -c "tso listcat | grep JENKINS"` works, so I guess this will too.